ccdproc/image_collection.py

```diff
@@ -216,6 +223,13 @@ def files(self):
         """
         return self._files
 
+    @property
+    def ext(self):
+        """
+        str,The extension for which all 
```

Comment:
And the type doesn't really match what can be used. As far as I know it can be `str` but `int` or a `str, int` tuple.